docs/configuration/monitoring.md

```diff
@@ -12,32 +12,51 @@ we have all functionality available on Grafana.
 
 The simplest way to setup prometheus in your cluster is by using the
 [Prometheus-Operator](https://github.com/helm/charts/tree/master/stable/prometheus-operator)
-bundle.
+bundle. Start by adding the custom resources required for installing this bundle in your cluster.
 
 ```sh
+# NOTE: if installing on microk8s use `microk8s.kubectl` instead of `kubectl`
```

Comment:
I would place the *NOTE:* outside the code block because it will be more visible. Later I will try to find a way to format all note / warning / ... text so it is more user-friendly and looks nicer.  :)